Lib/test/libregrtest/worker.py

```diff
@@ -14,6 +14,9 @@
 
 
 USE_PROCESS_GROUP = (hasattr(os, "setsid") and hasattr(os, "killpg"))
+NEED_TTY = set('''
```

Comment:
why not just use a set literal?

```
NEED_TTY = {
    'test_ioctl',
}
```

Reply:
I included this change in a follow-up PR: https://github.com/python/cpython/pull/119728